Source/RimFridge.csproj

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please remove this file from the PR

Reply:
OK

Reply:
However remember to add the FridgeCache.cs back to the project